src/main/java/Bean.java

```diff
@@ -129,22 +129,18 @@ public static void printErrorMessage() {
                 INDENT + "To mark a task as undone: unmark [task number]");
     }
 
-    public static void main(String[] args) {
+    public static void processUserInput() {
         String userInput;
         Scanner in = new Scanner(System.in);
 
-        greet();
-
-        outerLoop:
         while (Task.getNumberOfTasks() < MAX_LIST_COUNT) {
             userInput = in.nextLine();
             String userCommand = extractCommand(userInput);
 
             switch (userCommand) {
             case "bye":
                 // To exit
-                break outerLoop;
-
+                return;
```

Comment:
Should the "return" statement be removed? Might be redundant after a "break" statement.